data/release/global.xml

```diff
@@ -35,6 +20,13 @@
     <verboseExceptions>false</verboseExceptions>
     <metadata>
       <map>
+        <entry>
```

Comment:
@aaime , is the CogSettings change in the global.xml necessary for this PR? It causes an xstream exception when deploying a 2.28.x snapshot geoserver.war with the embedded data directory.

Reply:
No, it should not be an can be reverted.  At the same time, xstream should be setup to ignore unrecognized elements, so that's also an issue.